index.html

```diff
@@ -187,19 +190,17 @@
           </div>
           <!-- second part -->
           <div
-            class="grid grid-cols-2 grid-rows-2 sm:grid-rows-none sm:grid-cols-4 gap-3 p-2.5 border-outline border-button rounded-xl bg-color-all"
+            class="grid grid-cols-2 grid-rows-2 sm:grid-rows-none sm:grid-cols-4 gap-3 p-2.5 outline outline-[4px] outline-solid outline-[#1a1a1a] border border-[#262626] rounded-xl bg-[#141414] z-50"
```

Comment:
default outline is solid!!

Reply:
I have given it so that the new people can understand